package.json

```diff
@@ -16,7 +16,7 @@
     "bundle-report:assets": "WEBPACK_ANALYZE=true yarn build:assets",
     "bundle-report:assets:novo": "WEBPACK_ANALYZE=true yarn build:assets:novo",
     "bundle-report:server": "WEBPACK_ANALYZE=true yarn build:server",
-    "clean": "rm -rf .cache && rm -f manifest.json && rm -f manifest-novo.json && rm -rf public && mkdir public && echo '[Force] Cleaned build directory'",
+    "clean": "rm -rf .cache && rm -f manifest.json && rm -f manifest-novo.json && rm -rf public && mkdir public && rm -f src/mobile/public/assets/* src/desktop/public/assets/* && echo '[Force] Cleaned build directory'",
```

Comment:
Can we create a file in `/scripts` called `clean.sh` and call it from here?

Reply:
Yeah good call, this command is getting long